<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogQuestion</class>
 <widget class="QWidget" name="DialogQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="QuestionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>681</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color:white}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#000000;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stats">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Timer_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushAns0">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="autoRepeatDelay">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PushAns1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="autoRepeatDelay">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PushAns2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="LinkVisited">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>251</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="LinkVisited">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>251</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="LinkVisited">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>251</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="autoRepeatDelay">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PushAns3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="autoRepeatDelay">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QLabel" name="Background_Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>791</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Background_Label</zorder>
  <zorder>QuestionLabel</zorder>
  <zorder>Exit</zorder>
  <zorder>ErrorLabel</zorder>
  <zorder>Stats</zorder>
  <zorder>Timer_Label</zorder>
  <zorder>PushAns0</zorder>
  <zorder>PushAns1</zorder>
  <zorder>PushAns2</zorder>
  <zorder>PushAns3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
